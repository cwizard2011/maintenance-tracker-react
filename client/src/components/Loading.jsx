import React from 'react';

const Loading = () => (
  <div>
    <img src="images/loader.gif" className="text" alt="Loading..." />
  </div>
);

export default Loading;
