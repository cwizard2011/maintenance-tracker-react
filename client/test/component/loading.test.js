import React from 'react';
import { shallow } from 'enzyme';
import Loading from '../../src/components/Loading';

describe('Loader', () => {
  it('render loader correctly', () => {
    const wrapper = shallow(<Loading />);
    expect(wrapper).toMatchSnapshot();
  });
});
