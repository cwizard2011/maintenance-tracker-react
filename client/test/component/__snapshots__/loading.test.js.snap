// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader render loader correctly 1`] = `
<div>
  <img
    alt="Loading..."
    className="text"
    src="https://res.cloudinary.com/cwizard/image/upload/v1540768548/loader.gif"
  />
</div>
`;
